README.md

```diff
@@ -8,7 +8,7 @@ To create a new release of funding-service-design-utils:
 1. Make and test your changes as normal in this repo
 2. Update the `version` tag in `setup.py`
 3. Push your changes to `main`.
-4. The Action at `.github/workflows/create-tag-workflow.yml` will create a new tag and release, named for the version in `setup.py`. This is triggered automatically on a push to main.
+4. The Action at `.github/workflows/test-and-tag.yml` will create a new tag and release, named for the version in `setup.py`. This is triggered automatically on a push to main.
```

Comment:
Can we document how to run the tests locally?

Reply:
Done